bitnami/kube-prometheus/templates/prometheus/prometheus.yaml

```diff
@@ -45,6 +45,10 @@ spec:
   {{- if .Values.prometheus.image }}
   image: {{ template "kube-prometheus.prometheus.image" . }}
   {{- end }}
+{{- if .Values.prometheus.additionalArgs }}
+  additionalArgs:
+{{ toYaml .Values.prometheus.additionalArgs | indent 4}}
+{{- end -}}
```

Comment:
Could you fix the indentation here, to be consistent with the other lines? In addition, consider using `nindent` instead of `indent` (see other examples in the repo)

Reply:
Done